examples/ds-hostnet-split/03-contour.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - --xds-port=8001
         - --envoy-service-http-port=80
         - --envoy-service-https-port=443
-        - --contour-cafile=/ca/CAcert.pem
+        - --contour-cafile=/ca/cacert.pem
```

Comment:
thanks, mixed case will make you curse yourself when you mix windows and mac; CapsLock, just don't do it.